documentation/docs/index.html

```diff
@@ -0,0 +1,73 @@
+  <meta name="description" content="Axelar Testnet Docs">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="_style/light.css">
+  <link rel="stylesheet" href="_style/custom.css">
+  <link rel="stylesheet" href="_style/dark.css" media="(prefers-color-scheme: dark)">
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify-sidebar-collapse/dist/sidebar-folder.min.css" />
+</head>
+<body>
+  <div id="app"></div>
+</body>
+<script src="//cdn.jsdelivr.net/npm/docsify-edit-on-github/index.js"></script>
+
+<script>
+  window.$docsify = {
+    name: 'Axelar Testnet Docs',
+    auto2top: true,
+    logo: './axelar.svg',
+    coverpage: true,
+    notFoundPage: './_404.md',
+    onlyCover: true,
+    executeScript: false,
+    loadSidebar: true,
+    loadNavbar: true,
+    nameLink: 'https://axelar.network/',
+    notFoundPage: true,
+    // themeColor: '#3F51B5',
+    mergeNavbar: false,
+    maxLevel: 4,
+    subMaxLevel: 2,
+    search: {
+      noData: {
+        '/en/': 'No results :(',
+        '/': 'No results :('
+      },
+      paths: 'auto',
+      placeholder: {
+        '/en/': 'Search',
+        '/': 'Search'
+      }
+    },
+    formatUpdated: '{MM}/{DD} {HH}:{mm}',
+    externalLinkTarget: '_blank'
+  }
+
+// get all language from here: https://unpkg.com/browse/prismjs@1.25.0/components/
+</script>
+<script src="//cdn.jsdelivr.net/npm/docsify/lib/docsify.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/docsify/lib/plugins/search.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/docsify-pagination/dist/docsify-pagination.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-javascript.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-json.js"></script>
+<script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-jq.js"></script>
+<script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-json5.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-bash.min.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-yaml.min.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-yaml.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-log.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-vim.min.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-log.min.js"></script>
+<script src="//unpkg.com/prismjs/components/prism-toml.min.js"></script>
+<script src="//unpkg.com/docsify/lib/plugins/emoji.min.js"></script>
+<script src="//unpkg.com/docsify-copy-code"></script>
+<script src="https://cdn.jsdelivr.net/npm/docsify-example-panels"></script>
+<script src="//cdn.jsdelivr.net/npm/prismjs/components/prism-markdown.min.js"></script>
+<script src="//cdn.jsdelivr.net/npm/docsify-sidebar-collapse/dist/docsify-sidebar-collapse.min.js"></script>
```

Comment:
Not sure about all this script injection.  Is this necessary?  @canhtrinh

Reply:
Hi @ggutoski so each of these script adds a functionality - to give you an example:
```
<script src="//cdn.jsdelivr.net/npm/docsify/lib/plugins/search.min.js"></script>
```
provides the search fucntionality
```
<script src="//unpkg.com/prismjs/components/prism-*.min.js"></script>
```
anything starting with `prism` provide language highlighting - which I have added to highlight the code

```
<script src="//unpkg.com/docsify-copy-code"></script>
```
this allows a code block to be each copied by `copy to clipboard` button

```
<script src="//unpkg.com/docsify/lib/plugins/emoji.min.js"></script>
```
this turns `:bulb:` into 💡  and other emojis 
```
<script src="//cdn.jsdelivr.net/npm/docsify-sidebar-collapse/dist/docsify-sidebar-collapse.min.js"></script>
```
this allows a dropdown in sidebar and etc etc